src/main/java/com/rocketden/main/game_object/GameNotification.java

```diff
@@ -1,19 +1,20 @@
 package com.rocketden.main.game_object;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.rocketden.main.exception.GameError;
-import com.rocketden.main.exception.api.ApiException;
+import java.time.LocalDateTime;
 
-public enum GameNotification {
-    SUBMIT_CORRECT, SUBMIT_INCORRECT, TEST_CORRECT, CODE_STREAK, ONE_MIN_REMAINING;
+import com.rocketden.main.model.User;
 
-    // Convert a matching string (ignoring case) to enum object
-    @JsonCreator
-    public static GameNotification fromString(String value) {
-        try {
-            return GameNotification.valueOf(value.toUpperCase());
-        } catch (IllegalArgumentException e) {
-            throw new ApiException(GameError.BAD_SETTING);
-        }
-    }
+public class GameNotification {
```

Comment:
Is this class used anywhere or is it mainly the DTO? Mainly curious - I'm fine keeping it either way tho since we might need it later anyways

Reply:
Yeah good question, I put this in the description:

> The GameNotification class is never actually used -- only GameNotificationDto. Should I keep the GameNotification class?

I'm open to whatever for the same reasoning. I don't think we've ever actually just had the DTO version of the object, but it might make sense if we never use `GameNotification`.

Reply:
Same here - I'll leave it up to your discretion then

Reply:
Ok, I'll actually remove the `GameNotification` class, since it's not really doing anything.